modules/installation-replacing-control-plane-nodes.adoc

```diff
@@ -0,0 +1,300 @@
+:_mod-docs-content-type: PROCEDURE
+[id="installation-replacing-control-plane-nodes_{context}"]
+= Replacing control plane nodes in a two-node OpenShift cluster with fencing
+
+You can replace a failed control plane node in a two-node OpenShift cluster. The replacement node must use the same host name and IP address as the failed node.
+
+.Prerequisites
+
+* You have a functioning survivor control plane node.  
+* You have verified that either the machine is not running or the node is not ready.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You know the host name and IP address of the failed node.
+
+[NOTE]
+====
+It is recommended to take an etcd backup before proceeding to ensure that you can restore the cluster if any issues occur.
+====
+
+.Procedure
+
+. Check the quorum state by running the following command:
++
+[source,terminal]
+----
+$ sudo pcs quorum status
+----
++
+.Example output
+[source,terminal]
+----
+Quorum information
+------------------
+Date:             Fri Oct  3 14:15:31 2025
+Quorum provider:  corosync_votequorum
+Nodes:            2
+Node ID:          1
+Ring ID:          1.16
+Quorate:          Yes
+
+Votequorum information
+----------------------
+Expected votes:   2
+Highest expected: 2
+Total votes:      2
+Quorum:           1  
+Flags:            2Node Quorate WaitForAll 
+
+Membership information
+----------------------
+    Nodeid      Votes    Qdevice Name
+         1          1         NR master-0 (local)
+         2          1         NR master-1
+----
+
+.. If quorum is lost and one control plane node is still running, restore quorum manually on the survivor node by running the following command:
++
+[source,terminal]
+----
+$ sudo pcs quorum unblock
+----
+
+.. If only one node failed, verify that etcd is running on the survivor node by running the following command:
++
+[source,terminal]
+----
+$ sudo pcs resource status etcd
+----
+
+.. If etcd is not running, restart etcd by running the following command:
++
+[source,terminal]
+----
+$ sudo pcs resource cleanup etcd
+----
++
+If etcd still does not start, force it manually on the survivor node, skipping fencing:
++
+[IMPORTANT]
+====
+Before running this command, ensure that the node being replaced is inaccessible. Otherwise, you risk etcd corruption.
+====
++
+[source,terminal]
+----
+$ sudo pcs resource debug-stop etcd
```

Comment:
Break out each command to a separate code block as per https://redhat-documentation.github.io/supplementary-style-guide/#commands-in-code-blocks